modulemd/include/private/modulemd-build-config.h

```diff
@@ -346,4 +346,21 @@ modulemd_build_config_validate (ModulemdBuildConfig *buildconfig,
  */
 ModulemdBuildConfig *
 modulemd_build_config_copy (ModulemdBuildConfig *self);
+
+/**
+ * modulemd_build_config_equals:
+ * @self_1: A #ModulemdBuildConfig object.
+ * @self_2: A #ModulemdBuildConfig object.
+ *
+ * Check if @self_1 and @self_2 are equal objects.
```

Comment:
Rather than "equal objects", can you explain that we're testing whether they have identical *contents*? "Equal objects" sounds to me like we're checking to see if they are the same instance.